challenges/unit-testing/writing-tests/remove-vowels/remove-vowels.js

```diff
@@ -1,29 +1,7 @@
-function removeVowels(word) {
-  let characters = word.split("");
-
-  let result = [];
-
-  characters.forEach(function (character) {
-    if (
-      character !== "a" &&
-      character !== "o" &&
-      character !== "i" &&
-      character !== "e" &&
-      character !== "u"
-    ) {
-      result.push(character);
-    }
+function removeVowels(words) {
```

Comment:
This function is taking an array as well, which makes remove-vowels/remove-vowels-in-array.js pointless. While the code itself is good, could you think of a better structuring?

Reply:
Sure!